docs/new-editor.md

```diff
@@ -112,13 +113,16 @@ completions are disabled in the configuration
 (`scalac.completions.enabled=false`, default value), then metals responds with
 an empty list of completion suggestions.
 
-Client side configuration options include
+Clients are also encouraged to implement this setting:
 
 * `serverVersion: String`: while metals is still under active development, it is
   recommended to allow end-users to easily configure the version of the metals
   server.
-* `scalafmt.onSave: Boolean`: if true, runs scalafmt on
-  [`textDocument/willSaveWaitUntil`][] client requests.
+
+Note: we recommend to implement the scalafmt.onSave: Boolean setting only if the
+editor doesn't already have a more general mechanism for hanlding formatting on
+save. For example, thi setting is not included in the Visual Studio Code
+extension, becuase that particular editor has a dedicated API for it.
```

Comment:
typos: thi**s**.

Overall I don't think this is

Reply:
In general I think this advice is kind of ahead of time. Excluding this option was a very VSCode-specific thing, it's not only about having a more general option, but also about being ebla to enable it per-language. See discussion in https://github.com/scalameta/metals/pull/235

Reply:
Copy-pasted blindly, fixed typos.

Reply:
I suggest to put it a bit differently: "note that this setting may overlap with your existing editor functionality" and probably link to #235?